bin/common-functions.sh

```diff
@@ -18,7 +19,11 @@ dl() {
     [ -f "$target" ] && return
     t=$(dirname "$target")
     tfile=$(mktemp "$t/.${0##*/}.XXXXXX") || return
-    wget "$url" -O "$tfile" &&
+    wget_quiet=
+    if [ ! -z $QUIET ]; then
+        wget_quiet="-q"
+    fi
+    wget $wget_quiet "$url" -O "$tfile" &&
```

Comment:
we can probably just use `--progress=dot:mega` or `--progress=dot:giga`,  giga is 1 dot per 1M rather than 1 dot per 1k.

http://paste.ubuntu.com/p/M2s5xYbT2s/